docs/tutorial/6-viewsets-and-routers.md

```diff
@@ -3,7 +3,6 @@
 REST framework includes an abstraction for dealing with `ViewSets`, that allows the developer to concentrate on modeling the state and interactions of the API, and leave the URL construction to be handled automatically, based on common conventions.
 
 `ViewSet` classes are almost the same thing as `View` classes, except that they provide operations such as `read`, or `update`, and not method handlers such as `get` or `put`.
-
```

Comment:
why?

Reply:
I think I erased that line by accident, will add it back in. Thanks!